service/fexservice/Exception.py

```diff
@@ -0,0 +1,31 @@
+"""
```

Comment:
Só renomearia para `exception.py` https://www.python.org/dev/peps/pep-0008/#package-and-module-names